lib/plugins/blocks.js

```diff
@@ -74,14 +74,14 @@ function inject (bot, { version }) {
 
   function delColumn (chunkX, chunkY) {
     const columnCorner = new Vec3(chunkX * 16, 0, chunkY * 16)
-    const key = columnKeyXZ(columnCorner.x, columnCorner.z)
+    const key = columnKeyXZ(columnCorner.chunkX || columnCorner.x, columnCorner.chunkZ || columnCorner.z)
```

Comment:
All versions that defines unload_chunk packet uses chunkX/chunkZ you do not need the ||

Also you need to fix the issue at the source: https://github.com/PrismarineJS/mineflayer/blob/master/lib/plugins/blocks.js#L262 (columnCorner is a Vec3, not the packet)

Reply:
Sorry, stupid mistake. My first serious PR.

Reply:
You did not need to close it, you can push more commit to the PR to fix it. The bug you found needs fixing :)